generate_mocks.sh

```diff
@@ -4,25 +4,25 @@ BASEDIR=$(pwd)
 
 echo "Making mocks..."
 
-cd $BASEDIR/code/go/0chain.net/core || exit
+cd $BASEDIR/0chain.net/core || exit
 mockery --output=../core/mocks --all
 
-cd $BASEDIR/code/go/0chain.net/miner || exit
+cd $BASEDIR/0chain.net/miner || exit
 mockery --output=../miner/mocks --all
 
-cd $BASEDIR/code/go/0chain.net/chaincore || exit
+cd $BASEDIR/0chain.net/chaincore || exit
 mockery --output=../chaincore/mocks --all
 
-cd $BASEDIR/code/go/0chain.net/conductor || exit
+cd $BASEDIR/0chain.net/conductor || exit
 mockery --output=../conductor/mocks --all
 
-cd $BASEDIR/code/go/0chain.net/sharder || exit
+cd $BASEDIR/0chain.net/sharder || exit
 mockery --output=../sharder/mocks --all
 
-cd $BASEDIR/code/go/0chain.net/smartcontract || exit
+cd $BASEDIR/0chain.net/smartcontract || exit
 mockery --output=../smartcontract/mocks --all
 
-cd $BASEDIR/code/go/0chain.net/chaincore/chain/state || exit
+cd $BASEDIR/0chain.net/chaincore/chain/state || exit
 mockery --name=StateContextI --output=../../../mocks --filename=state-context-i.go
```

Comment:
Why are we putting a second copy of  StateContextI.go here. I know its popular, but having two copies of the same file is usally bad.

Reply:
this is not my change, it is better to ask original creator of it @MurashovVen .